exercises/react/src/example.c

```diff
@@ -0,0 +1,211 @@
+#include <stdlib.h>
+#include "react.h"
+
+enum cell_kind {
+   kind_input,
+   kind_compute1,
+   kind_compute2,
+};
+
+struct child {
+   struct cell *cell;
+   struct child *next;
+};
+
+struct cb {
+   callback_id id;
+   callback f;
+   void *obj;
+   struct cb *next;
+};
+
+struct reactor {
+   struct child *first;
+   struct child *last;
+};
+
+struct cell {
+   struct reactor *reactor;
+   int value;
+   enum cell_kind kind;
+   struct cell *input1;
+   struct cell *input2;
+   compute1 compute1;
+   compute2 compute2;
+   struct cb *cb;
+   int callbacks_issued;
+};
+
+struct reactor *create_reactor()
+{
+   struct reactor *r = calloc(1, sizeof(struct reactor));
+   return r;
```

Comment:
Apparently not!

Holdover from the times when I was using `malloc` and manually setting fields.